tests/baselines/reference/APISample_linter.js

```diff
@@ -75,26 +75,26 @@ function delint(sourceFile) {
     delintNode(sourceFile);
     function delintNode(node) {
         switch (node.kind) {
-            case 187 /* ForStatement */:

```

Comment:
@CyrusNajmabadi, not sure what you mean by that comment. Since this sample is part of the baselines, if I don't update it i'll end up with test failures.